docs/modelsearch.rst

```diff
@@ -502,3 +539,10 @@ Allow all combinations of absorption and elimination rates:
 
     ABSORPTION(*)
     ELIMINATION(*)
+
+All covariate effects on parameters with IIV:
+
+.. code::
+
+    COVARIATE(@IIV, @CONTINUOUS, *)
+    COVARIATE(@IIV, @CATEGORICAL, CAT)
```

Comment:
Maybe an example of defining symbols? Something I was also thinking of is that it could be could to have a list example since lists have different syntax is Python and in R.